src/mavis/interval.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, start: int, end: Optional[int] = None, freq: int = 1, number_
 
         self.start = self.number_type(self.start)
         self.end = self.number_type(self.end)
+        if self.start == 0 and self.end == 1:
```

Comment:
Rather than changing the base interval class it would be safer to add this workaround in the vcf.py file.